tutorials/animation/animation_tree.rst

```diff
@@ -188,6 +188,47 @@ Transitions also have a few properties. Click any transition and it will be disp
 * *Priority* is used together with the ``travel()`` function from code (more on this later). Lower priority transitions are preferred when travelling through the tree.
 * *Disabled* toggles disabling this transition (when disabled, it will not be used during travel or auto advance).
 
+For better blending
+-------------------
+
+In Godot 4.0+, for blending results consistently, the property values to be blended always have a specific initial value.
+For example, in the case of two animations to be blended, if one animation has a property track and the other does not,
+the blended animation is calculated as if the latter animation had a property track with the initial value.
+
+When using Position/Rotation/Scale 3D tracks for Skeleton3D bones, the initial value is Bone Rest.
+For other properties, the initial value is ``0`` (exactly the value of ``zero()`` as Variant)
```

Comment:
```suggestion
(exactly the value of ``zero()`` as Variant)
```
Is this a neccessary detail? It IMO makes an already complicated topic/sentence more complex.

Reply:
@mhilbrunner Would you recommend that I remove the `zero()` statement?

It was meant to suggest that there are two initial values for Variant, init() and zero(), but that we need to use zero() for correct calculation, but it is okay to remove it.

Position/Rotation/Scale 3D tracks for bones use Bone Rest. In this case, Bone Rest is implicitly and fundamentally never set to 0 on import.

However, Property tracks for changing other properties may become 0 unless RESET animation is explicitly set.

Reply:
Removed mention about zero().